schema/RemoveOrganizationProjectMutation.re

```diff
@@ -0,0 +1,51 @@
+open Graphql_lwt;
+open GraphqlHelpers;
+
+type removeOrganizationProjectInput = {
+  organizationId: int,
+  projectId: int,
+};
+
+let removeOrganizationProjectInput =
+  Schema.Arg.(
+    obj(
+      "RemoveOrganizationProjectInput",
+      ~fields=[
+        arg("organizationId", ~typ=non_null(int)),
+        arg("projectId", ~typ=non_null(int)),
+      ],
+      ~coerce=(organizationId, projectId) =>
+      {organizationId, projectId}
+    )
+  );
+
+let payload =
+  Schema.(
```

Comment:
hmm we could have been reusing this right? otherwise we're are just overwriting the same type over and over again

Reply:
talking about ErrorPayload